content/post/086-ipfs-in-opera-for-android.md

```diff
@@ -0,0 +1,98 @@
+# IPFS in Opera for Android
```

Comment:
Reminder to make this the proper blog heading with title/author/etc. I think we should replace this title with that since we have the image to head the blog post.